config_test.json

```diff
@@ -19,6 +19,16 @@
         "format": "20060102"
       }
     },
+    {
+      "name": "range",
+      "rangeConfig": [
```

Comment:
I think the `README` will need updating for this as the JSON I put in there at the time was slightly different. Also, I added the text:

> Both min and max are inclusive.

Which of course is not not true  - because you can decide! Awesome! 👍